Makefile

```diff
@@ -0,0 +1,6 @@
+release:
+	scripts/release.sh
+
+prep-release:
+	scripts/prep-release.sh
+	
```

Comment:
Let's make sure all the files have a proper new line, for POSIX compliance.

Reply:
There are also other files that do not, please have a look over those too.

Reply:
Done!